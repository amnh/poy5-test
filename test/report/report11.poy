(* Build report trees with parsimony branches; static *)
set(seed:50, timer:0)
read ("30.ss")
build (2)
report (trees:(branches:min))
report (trees:(branches:max))
report (trees:(branches:single))
