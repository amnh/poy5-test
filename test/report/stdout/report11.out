(t30:13,((t17:0,t21:8):12,(t24:10,(((t41:0,(t42:0,t43:0):0):4,((t18:0
,t22:0):2,((t31:0,t44:0):7,(t20:0,(t19:22,t23:2):0):3):4):2):8,((t32:42
,((t10:3,t9:12):9,(t36:2,t8:0):13):23):18,((t4:23,((t6:0,(t46:0,t49:0):0):20
,(t33:14,(t5:0,(t34:0,t7:1):0):11):21):13):26,(((t11:22,t37:40):13,((t14:7
,(t12:2,t40:2):7):20,((t13:12,t48:9):10,(t47:0,(t38:0,t39:0):0):19):20):19
):25,((t3:72,(t28:52,t29:37):30):32,(((t15:10,t45:9):4,(t16:4,t35:5):2):24
,((t1:14,t2:13):38,(t27:19,(t25:0,t26:0):35):39):24):27):31):25):22):20):11
):7):13)
;

(t30:13,((t17:0,t21:8):12,(t24:10,(((t41:0,(t42:0,t43:0):0):4,((t18:0
,t22:0):2,((t31:0,t44:0):7,(t20:0,(t19:22,t23:2):0):3):4):2):8,((t32:42
,((t10:3,t9:12):9,(t36:2,t8:0):13):23):18,((t4:23,((t6:0,(t46:0,t49:0):0):20
,(t33:14,(t5:0,(t34:0,t7:1):0):11):21):13):26,(((t11:22,t37:40):13,((t14:7
,(t12:2,t40:2):7):20,((t13:12,t48:9):10,(t47:0,(t38:0,t39:0):0):19):20):19
):25,((t3:72,(t28:52,t29:37):30):32,(((t15:10,t45:9):4,(t16:4,t35:5):2):24
,((t1:14,t2:13):38,(t27:19,(t25:0,t26:0):35):39):24):27):31):25):22):20):11
):7):13)
;

(t12:1,(t40:2,((t14:18,((t13:12,t48:9):9,(t38:0,(t39:0,t47:0):0):19):26):5
,(((t46:0,(t49:0,t6:0):0):20,(t4:24,(t33:16,(t34:0,(t5:0,t7:1):0):8):23):14
):37,((t11:20,t37:35):27,(((t24:9,(t30:29,(t17:0,t21:8):8):6):12,(t19:23
,(((t20:0,t23:2):1,(t31:0,t44:0):10):2,((t18:0,t22:0):2,(t41:0,(t42:0
,t43:0):0):2):5):6):6):25,((t32:43,((t10:1,t9:12):9,(t36:2,t8:0):11):21):22
,((t3:75,(t16:5,(t35:5,(t15:13,t45:8):5):7):35):14,((t1:14,t2:13):40,((t28:46
,t29:37):28,(t27:24,(t25:0,t26:0):32):37):27):31):32):19):33):13):34):18):1)
;

(t12:1,(t40:2,((t14:18,((t13:12,t48:9):9,(t38:0,(t39:0,t47:0):0):19):26):5
,(((t46:0,(t49:0,t6:0):0):20,(t4:24,(t33:16,(t34:0,(t5:0,t7:1):0):8):23):14
):37,((t11:20,t37:35):27,(((t24:9,(t30:29,(t17:0,t21:8):8):6):12,(t19:23
,(((t20:0,t23:2):1,(t31:0,t44:0):10):2,((t18:0,t22:0):2,(t41:0,(t42:0
,t43:0):0):2):5):6):6):25,((t32:43,((t10:1,t9:12):9,(t36:2,t8:0):11):21):22
,((t3:75,(t16:5,(t35:5,(t15:13,t45:8):5):7):35):14,((t1:14,t2:13):40,((t28:46
,t29:37):28,(t27:24,(t25:0,t26:0):32):37):27):31):32):19):33):13):34):18):1)
;

(t30:13,((t17:0,t21:0):0,(t24:0,(((t41:0,(t42:0,t43:0):0):0,((t18:0,t22:0):0
,((t31:0,t44:0):0,(t20:0,(t19:0,t23:0):0):0):0):0):0,((t32:0,((t10:0,t9:0):0
,(t36:0,t8:0):0):0):0,((t4:0,((t6:0,(t46:0,t49:0):0):0,(t33:0,(t5:0,(t34:0
,t7:0):0):0):0):0):0,(((t11:0,t37:0):0,((t14:0,(t12:0,t40:0):0):0,((t13:0
,t48:0):0,(t47:0,(t38:0,t39:0):0):0):0):0):0,((t3:0,(t28:0,t29:0):0):0
,(((t15:0,t45:0):0,(t16:0,t35:0):0):0,((t1:0,t2:0):0,(t27:0,(t25:0,t26:0):0
):0):0):0):0):0):0):0):0):0):13)
;
(t12:1,(t40:0,((t14:0,((t13:0,t48:0):0,(t38:0,(t39:0,t47:0):0):0):0):0
,(((t46:0,(t49:0,t6:0):0):0,(t4:0,(t33:0,(t34:0,(t5:0,t7:0):0):0):0):0):0
,((t11:0,t37:0):0,(((t24:0,(t30:0,(t17:0,t21:0):0):0):0,(t19:0,(((t20:0
,t23:0):0,(t31:0,t44:0):0):0,((t18:0,t22:0):0,(t41:0,(t42:0,t43:0):0):0):0):0
):0):0,((t32:0,((t10:0,t9:0):0,(t36:0,t8:0):0):0):0,((t3:0,(t16:0,(t35:0
,(t15:0,t45:0):0):0):0):0,((t1:0,t2:0):0,((t28:0,t29:0):0,(t27:0,(t25:0
,t26:0):0):0):0):0):0):0):0):0):0):0):1)
;

